ui/src/data-services/models/taxalist.ts

```diff
@@ -0,0 +1,64 @@
+import { getFormatedDateString } from 'utils/date/getFormatedDateString/getFormatedDateString'
+import { UserPermission } from 'utils/user/types'
+
+export type ServerTaxaList = {
+  id: number
+  name: string
+  description?: string
+  taxa: string // URL to taxa API endpoint (filtered by this TaxaList)
+  projects: number[] // Array of project IDs
+  created_at?: string
+  user_permissions?: UserPermission[]
+}
+
+export class TaxaList {
```

Comment:
If we extend class `Entity` we can simply this class a lot! I think `taxaUrl` is the only custom field.